### PR TITLE
rename last resolution scale option "native"->"maximum", DRS tweak

### PR DESCRIPTION
**Pull request by rfomin**
Like in Kex port, there might be less confusion.

### CODE REVIEW

File: src/mn_setup.c [hunk 1]
```diff
@@ -2105,7 +2105,7 @@ static const char **GetResolutionScaleStrings(void)
 
     resolution_scale = BETWEEN(0, i, resolution_scale);
 
-    array_push(strings, "native");
+    array_push(strings, "maximum");
```
[the remark this answers is not in the document]
Comment (rfomin) [reply src/mn_setup.c:2108]: Yes.
woof0002

**Review by ceski-1 — APPROVED** [review]

File: src/mn_setup.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (ceski-1) [reply src/mn_setup.c:2108]: Thanks for checking!

[the remark this answers is not in the document]
Comment (fabiangreffrath) [reply src/mn_setup.c:2108]: Else "max" should do as well.

[the remark this answers is not in the document]
Comment (ceski-1) [reply src/mn_setup.c:2108]: "Max" is probably better, yes.

[the remark this answers is not in the document]
Comment (rfomin) [reply src/mn_setup.c:2108]: Done in https://github.com/fabiangreffrath/woof/commit/7acf4180272f6ee079199013a8da9c59730545a1

**Pull request merged**